models/lobby.go

```diff
@@ -73,8 +76,23 @@ func (lobby *Lobby) Add(steamid string, name string, team int, slot int) error {
 }
 
 //Remove player from lobby
-func (lobby *Lobby) Remove(steamid string) {
-	delete(steamLobbyMap, steamid)
-	steamPlayerMap[steamid] = &Player{SteamId: "", Name: "id"}
-	delete(steamPlayerMap, steamid)
+func (lobby *Lobby) Remove(id string) {
+	delete(dbLobbyMap, id)
+}
+
+//Return an array of all lobbies
+func GetLobbyList(open bool) {
+	arr := make([]*Lobby, len(lobbyMap))
+
+	i := 0
+	for _, lobby := range lobbyID {
```

Comment:
1) Where does lobbyID come from? I can't find it in this file
2) Currently this function just copies lobbyID. Is that what you meant to do?
3) You should use slices instead of arrays. `var arr []*Lobby; for _, val := range oldArr { append(arr, val) }`